vipps-recurring-api.md

```diff
@@ -699,8 +700,8 @@ the charge states returned by
 
 | # | State      | Description                                                                          |
 |:--|:-----------|:-------------------------------------------------------------------------------------|
-| 1 | `PENDING`  | The charge has been created, but has not yet been approved by the user. |
```

Comment:
No idea what was meant by this. the user does not approve charges, and has never done.